bin/tessel-2.js

```diff
@@ -203,8 +203,13 @@ makeCommand('run')
     default: false,
     help: 'Deploy a project containing all files within, including those not used by the program, excluding any files matched by non-negated rules in .tesselignore and including any files matched by rules in .tesselinclude. Program is started from specified file.'
   })
+  .option('rustCC', {
+    // TODO: make this point to a long-lived server somewhere
+    default: 'http://localhost:8080',
+    help: 'Specify the location and port of the Rust cross-compilation server.'
+  })
   .help(`
-    Deploy a script to Tessel and run it with Node.
+    Deploy a script to Tessel and run it.
```

Comment:
Is this accurate? I think it should probably say "Compile and deploy a Rust program" (or something like that?